pom.xml

```diff
@@ -48,7 +48,13 @@
     <dependency>
       <groupId>com.google.api</groupId>
       <artifactId>gax</artifactId>
-      <version>1.57.1</version>
+      <version>1.56.0</version>
```

Comment:
We need an older version of this?

Reply:
Yeah, the newer version causes a conflict since vision and natural language depend on 1.56.0.